src/components/email-box/email-box.tsx

```diff
@@ -14,17 +14,20 @@ export class EmailBox {
   private readonly enterCode = 'Enter';
 
   @Prop()
-  field!: Field;
+  public field!: Field;
 
   @Prop()
-  control!: any;
+  public control!: any;
+
+  @Prop()
+  public readOnly: boolean;
```

Comment:
please add a TODO comment explaining the workaround for both properties. Future dev's can be confused of why we have separate props when field has everything internally, and we can find anyway in the future to deal with differently.

Reply:
Done :)